.cargo/config.toml

```diff
@@ -18,8 +18,10 @@ wasm-build-release = "panic-abort-build --target wasm32-unknown-unknown --profil
 
 [target.wasm32-unknown-unknown]
 rustflags = [
-  # 60 KiB, working around a bug in older rustc
+  # 100 KiB, working around a bug in older rustc
   # https://github.com/unicode-org/icu4x/issues/2753
   # keep in sync with wasm's build.sh
-  "-C", "link-args=-z stack-size=100000"
+  "-C", "link-args=-z stack-size=100000",
```

Comment:
Observation: there is a space between the `z` and the `stack` here but not in build.sh

Reply:
yeah, it doesn't matter, and it's easier to do it without the space in build.sh since it doesn't need to be quoted